packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -13,9 +13,15 @@ var get = Ember.get;
 
   ```javascript
   export default DS.Adapter.extend(BuildURLMixin, {
-    find: function(store, type, id, record) {
-      var url = this.buildURL(type.typeKey, id, record);
-      return this.ajax(url, 'GET');
+    createRecord: function(store, type, record) {
+      var data = {};
+      var serializer = store.serializerFor(type.typeKey);
+      var snapshot = record._createSnapshot();
+      var url = this.buildURL(type.typeKey, null, snapshot, 'createRecord');
```

Comment:
The code for `createRecord` pass `'create'` to buildURL, but in this example it's `'createRecord'`, typo?